### PR TITLE
Do not report the change until it is committed

### LINKED ISSUE #1158: Japanese input bug keyboard and flick input
From Twitter:

>My env is iOS 11.2 on iPhone 6.
>There is flic-input bag in Japanese on new toggl iOS app, I hope it will be fixed as soon as possible.
>For example, when typing with ABC, it is output as AABABC.(Replace the alphabet with Japanese characters)
>As I confirmed it now, the same problem has occurred not only in flick input but also in keyboard input in Japanese characters." 

https://twitter.com/mtane0412/status/940784885521895424/video/1

Assuming it's something like this? https://www.imore.com/how-use-keyboard-flick-ipads-quicktype-keyboard-ios-11

### PR DESCRIPTION
**Pull request by simonrozsival**
Closes #1158, Ref #1124 

This PR needs a functional review from someone, who is familiar with typing in Japanese on a smartphone.

### CODE REVIEW

File: Toggl.Daneel/Binding/TextViewTextInfoTargetBinding.cs [hunk 1]
```diff
@@ -138,8 +138,11 @@ public TagDeletedEventArgs(nint cursorPosition, nint tagIndex)
 
             [Export("textViewDidChange:")]
             public void DidChange(UITextView textView)
-                => TextChanged.Raise(this);
-           
+            {
+                if (textView.MarkedTextRange == null)
```
Comment (willsb) [inline Toggl.Daneel/Binding/TextViewTextInfoTargetBinding.cs]: Please early return when not null of use brackets. This looks confusing.

**Comment by amulware** [comment]
Also please add a comment for the reasoning behind the condition. This may not be obvious anymore in the future.

**Comment by willsb** [comment]
Question: Even though we don't report the text change, isn't the position change being reported somehow? Just asking to make sure we are not doing the calculations anyway because of a `cursor position changed ` event.

**Comment by simonrozsival** [comment]
I will test what happens when the the cursor moves.

**Comment by simonrozsival** [comment]
Changing the cursor while typing => commit.
Typing before committing => no event.

**Pull request merged**